### PR TITLE
Updates to gordon keymaps

### PR DESCRIPTION
**Pull request by DanielGGordon**
* Moved tap dance enums to gordon.h
    * Moved tap dance aliases to gordon.h
    * Moved TD to user space
    * Added config file with preventing mods sticking
    * Added a few keys to keymaps

Lots of things left to do, like using the new raise/lower feature, more LED customizaton, and updates to chimera layout.

### CODE REVIEW

File: users/gordon/gordon.c [hunk 1]
```diff
@@ -15,20 +15,17 @@ const char secret[][64] = {
...
   register_code (KC_LSFT);
-  register_code (KC_LCTL); 
-  register_code (KC_LALT); 
-  register_code (KC_LGUI); 
+  register_code (KC_LCTL);
+  register_code (KC_LALT);
+  register_code (KC_LGUI);
 }
 void unregister_hyper (void) { //Helper function to invoke Hyper
   unregister_code (KC_LSFT);
```
Comment (drashna) [inline users/gordon/gordon.c]: You could just call `clear_mods()` to clear all the mods. :)

File: keyboards/ergodox_infinity/keymaps/gordon/keymap.c [hunk 2]
```diff
@@ -10,266 +10,333 @@
...
+//        It is simply a visual reminder not to use that key for this layer.
+//        Example: On the numpad layer, under the letter `D` is `_XXXXXXX_`, because pressing and holding
+//          `D` sends you to the numpad layer
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
 
-//**************************FIRST LAYER - LAYER ZERO ************************************** 
+//**************************FIRST LAYER - LAYER ZERO **************************************
   [_QWERTY] = KEYMAP(
```
Comment (drashna) [inline keyboards/ergodox_infinity/keymaps/gordon/keymap.c]: Whaaaaa! You're not use `LAYOUT_ergodox_pretty`?!?!

**Comment by drashna** [comment]
Nice, thanks!

**Comment by DanielGGordon** [comment]
Ha I know it was about time. I still need to update chimera keymap and also add Orthodox!

**Comment by drashna** [comment]
haha, np :)

**Pull request merged**